packages/cryptodbm/cryptodbm.0.84.2/opam

```diff
@@ -0,0 +1,9 @@
+opam-version: "1.2"
+maintainer: "Didier Le Botlan <github.lebotlan@dfgh.net>"
+authors: "Didier Le Botlan <github.lebotlan@dfgh.net>"
+homepage: "https://github.com/lebotlan/ocaml-cryptodbm"
+bug-reports: "https://github.com/lebotlan/ocaml-cryptodbm/issues"
+license: "MIT License"
+dev-repo: "git://github.com/lebotlan/ocaml-cryptodbm.git"
+build: ["jbuilder" "build" "-p" name "-j" jobs]
+depends: ["ocamlfind" "jbuilder" "dbm" "fileutils" "cryptokit"]
```

Comment:
Please set a minimum OCaml version.

Also, do you really need an ocamlfind dependency here? If you use jbuilder as your build system you aren't actually make use of findlib.

Reply:
Thank you for reviewing the submission!

 - I have to set my minimum OCaml version to jbuilder's minimum ocaml version (that is 4.02.3).
 - Having switched from ocamlbuild --use-ocamlfind to jbuilder, I no longer need ocamlfind, indeed.
 - Thank you also for the automatic pieces of advice sent by email (they are automatic, right?)

 I'll close this pull request and resubmit soon.